src/NuGet.Clients/PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -576,6 +576,12 @@ private object GetVSSolutionProperty(int propId)
         {
             Debug.Assert(ThreadHelper.CheckAccess());
 
+            // if VSSolutionManager Initialization fail, _vsSolution can be null.
+            if (_vsSolution == null)
```

Comment:
As discussed, it won't crash VS, this initializeAsync happens after MEF component and NuGet packages are being initialized and you start a NuGet operation.